<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="trading_result">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>40</y>
      <width>141</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPlainTextEdit{
color:black;
border:1px solid;
}
</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_1">
    <property name="geometry">
     <rect>
      <x>564</x>
      <y>383</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:1px solid;
border-radius:15px;
color:black;
}

QPushButton:hover
{
background-color: white;

}
QPushButton:pressed
{
background-color:rgb(204, 204, 204);

}</string>
    </property>
    <property name="text">
     <string>Buy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>664</x>
      <y>383</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:1px solid;
border-radius:15px;
color:black;
}

QPushButton:hover
{
background-color: white;

}
QPushButton:pressed
{
background-color:rgb(204, 204, 204);

}</string>
    </property>
    <property name="text">
     <string>Sell</string>
    </property>
   </widget>
   <widget class="QLabel" name="labels">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>521</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color:black;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>591</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;</string>
    </property>
    <property name="text">
     <string>________________________________________________________________________________</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>true</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color:black;}</string>
    </property>
    <property name="text">
     <string>KAI</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_search">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:1px solid;
border-radius:15px;
color:black;
}

QPushButton:hover
{
background-color: white;

}
QPushButton:pressed
{
background-color:rgb(204, 204, 204);

}</string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_edit">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
color:black;
border:1px solid;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
